### PR TITLE
Add game event handlers

### PR DESCRIPTION
**Pull request by avrebarra**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -13,9 +16,12 @@ app.get('/dev/panel/', (req, res) => res.sendFile(path.join(__dirname, 'dev/test
 // setup socket server
 const io = socket(server)
 io.on('connection', socket => {
-  socket.on('ping', () => socket.emit('ping', 'pong'))
```
Comment (husnulhamidiah) [inline index.js:16]: let's keep this, we need it to display latency to user

File: handler.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import store from './store'
+
+export default {
+  handleCreateGame: (sock, data) => {
```
Comment (husnulhamidiah) [inline handler.js:4]: let's use `socket` for this params. easier to understand

File: handler.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import nanoid from 'nanoid'
+import store from './store'
+import memory from './memory'
+
+const genid = nanoid.customAlphabet('0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ', 8)
```
Comment (husnulhamidiah) [inline handler.js:5]: we can use native `crypto` module instead
```
crypto.randomBytes(4).toString('hex').toUpperCase()
```

File: handler.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    // generate things
+    const id = genid()
+    const room = {
+      id: id,
+      players: [],
+      rounds: []
+    }
+
+    // add current socket as player
+    const player = { id: genid(), isAdmin: true }
```
Comment (husnulhamidiah) [inline handler.js:20]: we can use `socket.id` for player id

Comment (avrebarra) [reply handler.js:24]: antisipasi dc?

Comment (avrebarra) [reply handler.js:24]: nek dc reconnect id e tetep?

**Pull request merged**